### ISSUE TITLE
[Feature Request] Option for blank prompts in SD3's triple encoder to create the same conditioning as when text encoder is absent, the same as the model was trained

### ISSUE THREAD

**Issue by CodeExplode** [comfyanonymous/ComfyUI#3785]
When SD3 is missing a text encoder, zeroes are passed instead. This seems to be how dropout was done during training, given the zeroing node of SD3's workflow. If T5 is absent, the prompt is not padded out for 77 zero tokens, rather just the first 77 CLIP token are passed (and vice versa).

I have also found that when finetuning with a particular conditioning (e.g. encoding blank prompts for the unconditional dropout, instead of zeroes), the model quickly adjusts to this, and then doesn't work with existing comfy workflows (e.g. zeroing). By the same logic, finetuning with only the CLIP models and no T5 would create a model which doesn't expect blank encoded T5 prompts as part of the conditioning, but rather requires the T5 to be handled as if it were completely absent.

Currently I load text encoders from the base SD3 checkpoint since there's no need to save them in my finetuning checkpoints if frozen, but that means there's no way to act as if T5 is missing as I train with. Having the option to zero each text encoder input (or rather, use the logic which comfy implements when it's absent altogether) would be more ideal.

edit: Sorry for the title changes, I somehow submitted the request while typing the description.

**Comment by comfyanonymous** [comment]
that's what the "empty_padding: none" option should do in CLIPTextEncodeSD3.

**Comment by CodeExplode** [comment]
Thanks you're correct, I saw that early on and didn't understand it at the time sorry.
